src/typings/RangoAction.ts

```diff
@@ -1,4 +1,5 @@
 interface RangoActionWithoutTargetWithoutArg {
+	arg: RangoActionWithTarget;
```

Comment:
What is this doing here? Voice coding accident?

Reply:
For us to script with an action we have  the scripting function and the function name that will be automatically run passed into it. So I think that this arg was referring to the specific Rango action that will operate on the reference (called a target in this instance)

That being said it might start to get confusing to have an action interface that takes an action as well so it maybe should be renamed to differentiate it from the other actions.